.env.example

```diff
@@ -1,2 +1,11 @@
 BASE_PATH=./
 PROJECT_NAME=dhii-php_template
+
+BUILD_ROOT_PATH=/app/
+
+PHP_BUILD_VERSION=7.2
+PHP_TEST_VERSION=7.2
+
+XDEBUG_VERSION=3.0.4
```

Comment:
Does this actually work for you? I still use xDebug 2.x everywhere, because I am sure it works. I'm very pro using 3.x, but gotta make sure it works for a Win setup first.

Reply:
Scratch tihis. It works for me.